net-im/skypeforlinux/skypeforlinux-5.1.0.1.ebuild

```diff
@@ -18,36 +20,36 @@ S="${WORKDIR}"
 QA_PREBUILT=opt/skypeforlinux/skypeforlinux
 RESTRICT="mirror bindist strip" #299368
 
-RDEPEND="dev-libs/atk
-	dev-libs/expat
-	dev-libs/glib:2
-	dev-libs/nspr
-	dev-libs/nss
-	gnome-base/gconf:2
+RDEPEND="dev-libs/atk[${MULTILIB_USEDEP}]
+	dev-libs/expat[${MULTILIB_USEDEP}]
+	dev-libs/glib:2[${MULTILIB_USEDEP}]
+	dev-libs/nspr[${MULTILIB_USEDEP}]
+	dev-libs/nss[${MULTILIB_USEDEP}]
+	gnome-base/gconf:2[${MULTILIB_USEDEP}]
 	gnome-base/libgnome-keyring
```

Comment:
I omitted this because libgnome-keyring doesn't have multilib support.

And libgnome-keyring is a hard dependency for Skype:
"Without the libgnome-keyring0 the application won't start. If you encounter this problem, make sure this package is installed."